builddeb.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM        ubuntu:trusty
+RUN         apt-get update && apt-get install -y --no-install-recommends \
+                build-essential \
```

Comment:
do you actually need all of `build-essential` -- you're not actually compiling anything?

Reply:
Probably not, but it looks like `dpkg` has a hard check for `build-essential`:
https://gist.github.com/ATRAN2/f3fca76938bade90d88a2fce8a831e22

I found the `--ignore-builtin-builddeps` flag and tried it out. It seems to pass it to `dpkg-buildpackage` correctly, but the flag wasn't found:
https://gist.github.com/ATRAN2/6ce0f1b01bf481a123f20422d03d7993

Reason being the trusty dpkg only goes up to `1.17.5ubuntu5.8` while the flag was added in `1.18.2`
```
root@d5ec8d4b2ee3:/# dpkg -l | grep dpkg
ii  dpkg                            1.17.5ubuntu5.8                            amd64        Debian package management system
ii  dpkg-dev                        1.17.5ubuntu5.8                            all          Debian package development tools
ii  libdpkg-perl                    1.17.5ubuntu5.8                            all          Dpkg perl modules
```

So I might be stuck with installing `build-essential`
Or I can build on xenial which seems to go up to `1.18.4ubuntu1.4`

Reply:
nevermind then, lol silly dpkg